packages/core/package.json

```diff
@@ -7,7 +7,6 @@
     "simplebar-react": "^2.1.0"
   },
   "peerDependencies": {
-    "@material-ui/core": "^4.9.3",
```

Comment:
If we don't included `@mui/material` here, how does core import `@mui/material` components?

Reply:
I agree. Let me add this.